src/components/SCAInfoIconWithPopover/SCAInfoIconWithPopover.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { FunctionComponent } from 'react';
 import { Popover } from '@patternfly/react-core';
-import InfoIcon from '@patternfly/react-icons/dist/js/icons/info-circle-icon';
+import { QuestionCircleIcon } from '@patternfly/react-icons/dist/js/icons/question-circle-icon';
```

Comment:
I guess this package exports the icon as default as well as named? Not a bad thing just curious

Reply:
Resolving based on offline conversation